IdentityCore/src/cache/key/MSIDDefaultAccountCacheQuery.m

```diff
@@ -35,16 +35,6 @@ - (NSString *)account
     return nil;
 }
 
-- (NSNumber *)type
-{
-    if (self.accountType != MSIDAccountTypeOther)
```

Comment:
Would it be safer to instead default to MSSTS? I'm concerned that we will have account query somewhere in our code that doesn't set this and will fail the query.

Reply:
Sure, although I cross-checked in MSAL and broker but I might missed other libs. I will update to your suggestion.

Reply:
Addressed